src/app/public/index.ts

```diff
@@ -1 +1,5 @@
 export * from './modules';
+
+export * from './modules/country-field/country-field.component';
```

Comment:
Do we need to export the component to the public API?

Reply:
Good call. Lets go with "no" and we can always add it back later.